### PR TITLE
Fix minor typo & command for documentation

### PR DESCRIPTION
**Pull request by itholic**
This PR proposes to fix minor typo and command for documentation as below:

**command**
- remove the command `cd py4j-python` in [Testing Python Code](py4j-web/contributing.rst), since `requirements.txt` is in the project root path, not in `/py4j-python`.

**typos**
- occured -> occurred
- Builded -> Built
- shutted -> shut

### CODE REVIEW

File: py4j-web/contributing.rst [hunk 1]
```diff
@@ -113,7 +113,6 @@ suite across the supported python versions.
   ./gradlew assemble
 
   # install test requirements
-  cd py4j-python
```
Comment (HyukjinKwon) [inline py4j-web/contributing.rst:116]: I think we should change this as `cd py4j-python` and `requirements.txt` -> `requirements-test.txt`

File: py4j-web/contributing.rst [hunk 2]
```diff
@@ -113,7 +113,6 @@ suite across the supported python versions.
   ./gradlew assemble
 
   # install test requirements
```
Comment (HyukjinKwon) [inline py4j-web/contributing.rst:115]: and I would add some comments to guide users to come back to the Py4J home directory, or add `cd ..`

Comment (itholic) [reply py4j-web/contributing.rst:115]: Sounds good. Let me address the comments, thanks!

**Review by HyukjinKwon — APPROVED** [review]

**Comment by HyukjinKwon** [comment]
@itholic mind retriggering the tests to make sure the test pass?

**Comment by itholic** [comment]
Sure

**Comment by itholic** [comment]
FYI: Seems like the patch is applied to the documentation properly.

Screen Shot 2022-03-10 at 11 03 50 AM

**Pull request merged**